### ISSUE TITLE
The .NET global tool version needs lzma.dll

### ISSUE THREAD

**Issue by ltrzesniewski** [xoofx/kalk#1]
Well, sorry, but it's time for the first issue 😅 

I tried to install kalk as a .NET tool, and here's the console output:

```
C:\Users\lucas>dotnet tool install --global kalk --version 0.1.0
You can invoke the tool using the following command: kalk
Tool 'kalk' (version '0.1.0') was successfully installed.

C:\Users\lucas>kalk
Error:
  An assembly specified in the application dependencies manifest (kalk.deps.json) was not found:
    package: 'Packaging.Targets', version: '0.1.189'
    path: 'runtimes/win7-x64/native/lzma.dll'
```

I have the .NET SDK installed through a VS 2019 build tools installation on this machine.

Thanks for this tool BTW! 😄

**Comment by xoofx** [comment]
doh, I should have checked it. This assembly (`Packaging.Targets`) should not be included. It's only used for the build.

**Comment by xoofx** [comment]
Ok, It should fixed with the new version `0.1.1`, sorry for the trouble!

```
dotnet tool install --global kalk --version 0.1.1
```
